### PR TITLE
fix(renderer): Reducing max cap for point lights to reduce shader reflection time on OSX.

### LINKED ISSUE #472: Long boot time on macOS
When you launch any amethyst driven program on any macOS system, the boot time (with the white screen) is very long compared to other OS, it's displayed more than 5 seconds.

### PR DESCRIPTION
**Pull request by Rhuagh**
Closes #472.

### CODE REVIEW

File: amethyst_renderer/src/pass/shaders/fragment/pbm.glsl [hunk 1]
```diff
@@ -17,7 +17,7 @@ struct PointLight {
 };
 
 layout (std140) uniform PointLights {
-    PointLight plight[512];
+    PointLight plight[128];
```
Comment (torkleyy) [inline amethyst_renderer/src/pass/shaders/fragment/pbm.glsl]: Would also be interesting to control this with some template substitution.

**Comment by Xaeroxe** [comment]
Hang on before we merge this, there might be a better way: https://www.khronos.org/opengl/wiki/Shader_Storage_Buffer_Object

**Comment by Rhuagh** [comment]
Not sure gfx support that and the limit will disappear when we go to hal.

**Comment by Xaeroxe** [comment]
:+1: merge away then

**Comment by Rhuagh** [comment]
And I'd say, the scenes that need > 128 active point lights will be kind of rare :P

**Pull request merged**